ImageCapture/webcam_image_capture.py

```diff
@@ -0,0 +1,46 @@
+import cv2
+import os
+import time
+from datetime import datetime
+
+# Paramètres
+interval_seconds = 5  # Intervalle de temps entre chaque capture en secondes
+save_directory = "captured_images"  # Dossier où les images seront enregistrées
+
+# Création du dossier pour stocker les images, s'il n'existe pas
+if not os.path.exists(save_directory):
+    os.makedirs(save_directory)
+
+# Initialisation de la caméra (0 pour la caméra par défaut, essayez 1 si vous utilisez une autre webcam)
+cap = cv2.VideoCapture(0)
+

```

Comment:
When I try it on my computer, the 5 first pictures are the same. 
You can reduce the size of the VideoCapture buffer by using this line `cap.set(cv2.CAP_PROP_BUFFERSIZE, 1)`.
It means that the two first images will be the same. To fix that I duplicated line 25 (` ret, frame = cap.read()
`) and now all pictures are taken on time.
Could you try that on your side please?